docs/components/datetimepicker.md

```diff
@@ -0,0 +1,407 @@
+# DateTime Picker
+
+<div v-if="hasMounted">
+
+**KDateTimePicker** - A date and time selection tool, displayed inside a popover.
+
+## Examples
+
+A single date or time picker instance will emit a Jasvscript Date object as a `string`.
+Whereas an instance of this component that displays a date range or a series of relative time frames, will emit an `object` containing the following values:
+
+```js
+{ 
+  "start": "2022-09-06T19:45:54.406Z",
+  "end": "2022-09-07T19:45:54.406Z",
+  "timePeriodsKey": "24h" // or ""
```

Comment:
For these examples, would it make sense to document the type of thing they point to instead of an example? (Or at least point out the type alongside the example). 

e.g. `Date()` or 'String` or whatever 

We should also show here what a single date looks like.